src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -974,6 +961,9 @@ export class MeetingManager implements AudioVideoObserver {
       callback(name, attributes)
     );
   };
+
+  get eventController (): EventController { return this._eventController }
```

Comment:
Please correct me if wrong: We internally create the `EventController` if not passed as it is optional in that case, should this getting provide back the created `eventController` and not only `this._eventController`? Like also check if we have a `meetingSession` and return the `eventController` from it? This also should handle the `undefined` case right?